tests/generation/test_logits_process.py

```diff
@@ -1391,3 +1403,148 @@ def check_eos_logits(out, logits, batch, channel, eos):
         check_eos_logits(out=out, logits=logits, batch=1, channel=0, eos=eos)
         self.assertTrue(delay_pattern_processor.active_batches.all())
         self.assertTrue((delay_pattern_processor.delay_pattern == torch.tensor(delay_pattern) - 1).all())
+
+class TestLogitProcessorRegistry(unittest.TestCase):
```

Comment:
Let's have a single class for everything related to the configurable logits processors 🤗